pkg/kubelet/cm/dra/plugin/plugins_store.go

```diff
@@ -74,3 +105,16 @@ func (s *PluginsStore) Clear() {
 
 	s.store = make(map[string]*Plugin)
 }
+
+func newGrpcConn(addr string) (*grpc.ClientConn, error) {
```

Comment:
This function seems to be called only once. I'd suggest to get rid of it and call `grpc.Dial` in `CreateGRPCConn`. The code would look more readable this way in my opinion.

Reply:
nice catch thanks :) done in 8baa57effbe